backend/src/main/java/planit/people/preparation/API_Documentation/snippets/calendar/create_preset/http-response.adoc

```diff
@@ -4,11 +4,14 @@ HTTP/1.1 201 Created
 Vary: Origin
 Vary: Access-Control-Request-Method
 Vary: Access-Control-Request-Headers
+Content-Type: application/json
 X-Content-Type-Options: nosniff
 X-XSS-Protection: 1; mode=block
 Cache-Control: no-cache, no-store, max-age=0, must-revalidate
 Pragma: no-cache
 Expires: 0
 X-Frame-Options: DENY
+Content-Length: 905
 
+{"event_preset":{"id_event_preset":1,"name":"Test 1","break_into_smaller_events":false,"min_length_of_single_event":null,"max_length_of_single_event":null,"shared_presets":[]},"guests":[{"id_event_guest":1,"entity_EventPreset":null,"email":"test@gmail.com","obligatory":true},{"id_event_guest":2,"entity_EventPreset":null,"email":"test2@gmail.com","obligatory":true},{"id_event_guest":3,"entity_EventPreset":null,"email":"test3@gmail.com","obligatory":true}],"preset_availability":[{"id_preset_availability":1,"entity_EventPreset":null,"day":"THURSDAY","start_available_time":null,"end_available_time":null,"day_off":false},{"id_preset_availability":2,"entity_EventPreset":null,"day":"MONDAY","start_available_time":null,"end_available_time":null,"day_off":true},{"id_preset_availability":3,"entity_EventPreset":null,"day":"SATURDAY","start_available_time":null,"end_available_time":null,"day_off":true}]}
```

Comment:
this exact string is also repeated multiple times, why can't we make it a variable?